tests/sass/space-around-operator.sass

```diff
@@ -333,6 +333,8 @@ $qux: (2  +1)
 
 $foo: scale-color($foo, $lightness: -14%)
 
+$foobar: #{$foo}-#{$bar}
```

Comment:
Not sure it's a working example but sassmeister parses this 
```scss
$bar: 'test';

.test {
  content: #{$bar}-4;
}
```
should probably check this isn't a error either.